website/src/routes/guides/(get-started)/comparison/index.mdx

```diff
@@ -1,6 +1,9 @@
 ---
 title: Comparison
-description: Even though my API resembles other solutions at first glance, the implementation and structure of the source code is very different.
+description: >-
```

Comment:
Whats the reason for this change?

Reply:
@fabian-hiller 
Sorry for my late reply 🙏 
This is because of the built-in formatter of stringify of [grey-matter](https://github.com/jonschlinkert/gray-matter/tree/master).
And it seems unable to change the format, because grey-matter uses [js-yaml](https://github.com/nodeca/js-yaml) internally to stringify, and cannot change the option of js-yaml, through grey-matter.

If you don't like this format, I'm gonna search other libraries 😄

Reply:
Isn't Qwik already able to read metadata with `useDocumentHead().frontmatter`? Do we even need an additional library for this?

Reply:
@fabian-hiller 
I think we need this library, as long as `pnpm contributors` (tsm contributors.ts) can be executed to add contributors in each docs.

By using `useDocumentHead().frontmatter`, Owik can only read metadata, but in contributors.ts, metadata about contributors is read and also written.

Should I delete contributors.ts? 👀